### PR TITLE
[#44] [Integration] As a user, I can close the survey at any time - iOS

### LINKED ISSUE #44: [Integration] As a user, I can close the survey at any time
## Why

While taking a survey, the user can close it at any time and do it later by tapping on the close button in the top right corner.

## Acceptance criteria

Show the default warning pop up:

- Title: `Warning!`
- Description: `Are you sure you want to quit the survey?`
- `Yes` and `Cancel` options

If the user chooses `Yes`, close the survey and navigation back to the home screen.

If the user chooses `Cancel`, dismiss the pop-up and continue answering the survey.

### PR DESCRIPTION
**Pull request by nmint8m**
- Close #44

## What happened 👀

In this PR, I implemented the coordinator for the iOS app.

Besides, I also implemented showing an alert and closing a survey.

For Android, I will implement in another PR.

## Insight 📝

Use the [FlowStacks](https://github.com/johnpatrickmorgan/FlowStacks) to implement app navigation.

## Proof Of Work 📹

The iOS app can navigate between screens

The user can close the survey after tapping Yes on the alert

### CODE REVIEW

File: ios/ios/Resources/Localizations/en.lproj/Localizable.strings [hunk 1]
```diff
@@ -1,3 +1,7 @@
+"cancel" = "Cancel";
 "home.header.today" = "Today";
 "surveyDetail.primaryButton.title" = "Start Survey";
 "question.submitButton.title" = "Submit";
+"question.quit.title" = "Warning!";
```
Comment (markgravity) [inline ios/ios/Resources/Localizations/en.lproj/Localizable.strings]: What do you think about making more clear here?
`question.quit.title` -> `question.quit_alert.title`

File: ios/ios/Resources/Localizations/en.lproj/Localizable.strings [hunk 2]
```diff
@@ -1,3 +1,7 @@
+"cancel" = "Cancel";
```
Comment (markgravity) [inline ios/ios/Resources/Localizations/en.lproj/Localizable.strings]: Please follow our rule on naming the localized key, for example
`dialog.cancel_button.title`

File: ios/ios/Sources/Presentation/Modules/Home/HomeContainerView.swift [hunk 3]
```diff
@@ -29,9 +33,13 @@ struct HomeContainerView: View {
             }
         }
     }
+
+    private func onTap(index: Int) {
+
```
Comment (markgravity) [inline ios/ios/Sources/Presentation/Modules/Home/HomeContainerView.swift]: Please add `//TODO` here 🙏

**Review by markgravity — COMMENTED** [review]
Please create a chore task next time for out-of-scope changes.
We should only work with Quit Alert in this task and create a chore task to set up coordinator, it will be easier for us to review 🙏

**Review by markgravity — APPROVED** [review]

**Review by thiennguyen0196 — APPROVED** [review]
Nice work 👏 lgtm

**Pull request merged**